ng-dev/release/notes/release-notes.ts

```diff
@@ -70,23 +69,17 @@ export class ReleaseNotes {
    * provided by the GitClient.
    */
   async prependEntryToChangelog() {
-    /** The changelog contents in the current changelog. */
-    let changelog = '';
-    if (existsSync(this.changelogPath)) {
-      changelog = readFileSync(this.changelogPath, {encoding: 'utf8'});
-    }
-    /** The new changelog entry to add to the changelog. */
-    const entry = await this.getChangelogEntry();
+    const changelog = new Changelog(this.git);
 
-    writeFileSync(this.changelogPath, `${entry}\n\n${changelog}`);
+    changelog.prependEntryToChangelog(await this.getChangelogEntry());
 
     // TODO(josephperrott): Remove file formatting calls.
     //   Upon reaching a standardized formatting for markdown files, rather than calling a formatter
     //   for all creation of changelogs, we instead will confirm in our testing that the new changes
     //   created for changelogs meet on standardized markdown formats via unit testing.
     try {
       assertValidFormatConfig(this.config);
-      await formatFiles([this.changelogPath]);
+      await formatFiles([this.changelog.changelogPath]);
```

Comment:
Optional idea: How about changing the constant to `this.changelog.filePath` and `this.changelog.archiveFilePath` 

(to avoid the repetition; the context of changelog is already clear within the class)